onnxruntime/core/mlas/lib/convsym.cpp

```diff
@@ -415,6 +399,27 @@ MlasConvSymDepthwise(
 
     MlasConvSymSetOutputZeroPoint(PostProcessParams, Params.OutputZeroPoint);
 
+#if defined(MLAS_TARGET_ARM64)
+
+    if ((Params.KernelSize == 9 || Params.KernelSize == 25) && (Params.OutputChannels & 15) == 0) {
+        PostProcessParams.Bias = Params.Bias;
+        PostProcessParams.Scale = Params.Scale;
+        if (Params.KernelSize == 9) {
+            MlasConvSymDepthwiseKernelSize9Arm64(
+                Params.InputIndirection, (int8_t const*)Params.Filter, Params.OutputChannels, Params.Output,
+                Params.OutputCount, &PostProcessParams, KernelFlags
+            );
+        } else {
+            MlasConvSymDepthwiseKernelSize25Arm(
+                Params.InputIndirection, (int8_t const*)Params.Filter, Params.OutputChannels, Params.Output,
+                Params.OutputCount, &PostProcessParams, KernelFlags
+            );
+        }
+        return;
+    }
+
+#endif
```

Comment:
could we add a wrapper function and merge it together with the general one?

Reply:
I think later will be better time.